src/components/NavTop.js

```diff
@@ -47,119 +48,35 @@ class NavTop extends Component {
               <ul className="nav navbar-nav navbar-right">
-
-                {/*<li className="dropdown">
-                  <a
-                  href=""
-                  className="dropdown-toggle"
-                  data-toggle="dropdown">
-                  <i className="material-icons">notifications</i>
-                  <span className="notification">5</span>
-                  <p className="hidden-lg hidden-md">
-                  Notifications
-                  <b className="caret" />
-                  </p>
-                  </a>
-                  <ul className="dropdown-menu">
-                  <li>
-                  <a href="">
-                  Mike John responded to your email
-                  </a>
-                  </li>
-                  <li>
-                  <a href="">
-                  You have 5 new tasks
-                  </a>
-                  </li>
-                  <li>
-                  <a href="">
-                  You're now friend with Andrew
-                  </a>
-                  </li>
-                  <li>
-                  <a href="">
-                  Another Notification
-                  </a>
-                  </li>
-                  <li>
-                  <a href="">
-                  Another One
-                  </a>
-                  </li>
-                  </ul>
-                  </li>*/}
-                  <li className=" ">
-                    <a href="" onClick={this.props.logoutUser}>
-                      <i className="material-icons">fingerprint
-                      </i> Logout
+                  <li className="">
+                    <a href="" onClick={() => {
+                      this.props.logoutUser();
+                    }}>
+                      <i className="material-icons">fingerprint</i> Logout
                     </a>
                   </li>
-                  <li className="dropdown">
-                    <a href=""
-                      onClick={() => this.props.history.push(`/${localStorage.getItem('username')}`)}
-                      className="dropdown-toggle"
-                      data-toggle="dropdown-profile">
+                  <li>
+                    <a href={`/profile/${localStorage.getItem('username')}`}>
                       <i className="material-icons">person</i>
                       <p className="hidden-lg hidden-md">Profile</p>
                     </a>
-                    <ul className="dropdown-menu">
-                      <li>
-                        <a href="">
-                          Mike John responded to your email
-                        </a>
-                      </li>
-                      <li>
-                        <a href="">
-                          You have 5 new tasks
-                        </a>
-                      </li>
-                      <li>
-                        <a href="">
-                          You're now friend with Andrew
```

Comment:
The apostrophe might break a linter.  We can use the entity `&apos;` instead.

Reply:
Nevermind.  You deleted this

Reply:
Yeah it wasn't being used, and we can always add it later if we want it back